tests/Admin/Controller/DashboardControllerTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Tests\Admin\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class DashboardControllerTest extends WebTestCase
+{
+    /**
+     * @dataProvider provideUrls
+     */
+    public function testPageIsSuccessful($url)
+    {
+        $client = self::createClient();
+        $client->catchExceptions(false);
+        $client->request('GET', $url);
+        $response = $client->getResponse();
+
+        self::assertTrue(
+            $response->isSuccessful(),
+            sprintf(
+                'La page devrait être accessible, mais le code HTTP est "%s".',
+                $response->getStatusCode()
+            )
+        );
```

Comment:
You can use [the assertions provided by Symfony](https://symfony.com/doc/current/testing.html#testing-application-assertions):

```suggestion
        $this->assertResponseIsSuccessful();
```

Also, the error messages should probably be written in English (except if the software targets only a French-speaking audience and French-speaking developers).